src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -134,6 +134,12 @@ The SDK sets the Span operation to `app.start.cold` for *Cold start_* and `app.s
 
 The SDK uses the `SentryPerformanceProvider` (ContentProvider) creation time as the beginning of the app start and the first `Activity#onResume` call as the end.
 
+<Note>
+
+The App Start Instrumentation is only measured if the process is of the importance [RunningAppProcessInfo.IMPORTANCE_FOREGROUND](https://developer.android.com/reference/android/app/ActivityManager.RunningAppProcessInfo#IMPORTANCE_FOREGROUND), which means the process is running the foreground UI.
```

Comment:
Is the app start instrumentation the actual thing being measured here? If so, then this is fine. If not, then you might need to clarify what is being measured. Or maybe "measured" isn't the right word.

Reply:
the App start is a "measurement" and it's measured since it's calculated from a start and end timestamp.
eg App start = 0.5s
I'm open for suggestions tho :)

Reply:
Ah okay. So then is this more accurate:
```suggestion
The app start is only measured if the process is of the importance [RunningAppProcessInfo.IMPORTANCE_FOREGROUND](https://developer.android.com/reference/android/app/ActivityManager.RunningAppProcessInfo#IMPORTANCE_FOREGROUND), which means the process is running the foreground UI.
```

Reply:
thanks as always @imatwawana